### PR TITLE
#7 Creating ETL Script

### PR DESCRIPTION
**Pull request by bentrzcinski**
- Created the full etl.py pipeline script
- Amended load.py to work for a dataframe instead of a csv
- Updated the pipeline requirements.txt file with appropriate libraries

### CODE REVIEW

File: pipeline/etl.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    logging.info("\n --- Starting Extract --- \n")
+    raw_data = main_extract()
+    logging.info("\n --- Extract Complete --- \n")
+    logging.info("\n --- Starting Transform --- \n")
+    processed_data = main_transform(raw_data)
+    logging.info("\n --- Transform Complete --- \n")
+    logging.info("\n --- Starting Load --- \n")
+    main_load(processed_data)
+    logging.info("\n --- Load Complete --- \n")
+    logging.info("\n --- Finished Pipeline Execution --- \n")
```
Comment (GabrielNsiah) [inline pipeline/etl.py:22]: Love the clear logging here!

**Review by GabrielNsiah — APPROVED** [review]

File: pipeline/etl.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import logging
+from extract import main_extract, config_log
+from transform import main_transform
+from load import main_load
+
+
+def main_etl():
+    """The main pipeline script that fully extract, transforms and loads the bandcamp data."""
+    config_log()
```
Comment (emily-curtis) [inline pipeline/etl.py:11]: Love the use of logging to tell a story!

**Review by emily-curtis — APPROVED** [review]
Looks great!

**Review by ebradley12 — COMMENTED** [review]
All looks good to me!

**Pull request merged**